src/Stryker.Core/Stryker.Core/MutantFilters/MutantFilterFactory.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Stryker.Core.DiffProviders;
+using Stryker.Core.Options;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stryker.Core.MutantFilters
+{
+    public static class MutantFilterFactory
+    {
+        public static IMutantFilter Create(StrykerOptions options)
+        {
+            if(options == null)
+            {
+                throw new ArgumentNullException($"{nameof(options)} cannot be null");
```

Comment:
Contrary to most other exception constructors, the first parameter of `ArgumentNullException` is not `message` but `paramName`.